### DISCUSSION TITLE
HLTV scraper

### DISCUSSION THREAD

**Discussion by chinaPancake** [ultrafunkamsterdam/undetected-chromedriver#1483] [General]
I've made whole application that is scraping data from hltv it was helping me avoid the CloudFlare but since 15.08.2023 it's not working anymore, i guess i could get ban from hltv or smth went wrong with the UC. 

For exmpale on my own pc code work as it should be, but on the server it's no logner working correctly. 

Anyone was trying to scrape hltv?

**Comment by Charlotte-br560** [comment]
Hey there! It sounds like you're experiencing some issues with scraping data from HLTV. Changes to their website or security measures like CloudFlare may have affected your scraper. While scraping can be tricky, you might want to check your server setup or consider using a tool like [Crawlbase](https://crawlbase.com/), designed for web scraping tasks and can help you avoid common pitfalls. Good luck!
